build.gradle

```diff
@@ -98,22 +104,6 @@ gradlePlugin {
     }
 }
 
-pluginBundle {
-    website = 'https://github.com/palantir/gradle-consistent-versions'
-    vcsUrl = 'https://github.com/palantir/gradle-consistent-versions'
-    description = 'Compact, constraint-friendly lockfiles for your dependencies'
-    tags = ['versions']
-
-    // Don't publish the other plugins to the plugin portal
-    plugins.removeIf { it.name != 'consistentVersions' }
```

Comment:
genuinely not sure what is the right substitute of this method

Reply:
Based https://docs.gradle.org/current/userguide/java_gradle_plugin.html#sec:gradle_plugin_dev_interactions what I have done I think does the same thing as this block